openmicroscopy/openmicroscopy
# Imagej prep

In this PR,
* Refactor import code to handle imagePlus object.
* Add 2 entries to the OMERO plugin menu.

To Test:
 * make sure you can still import images using the standard build.

ImageJ Test
 * Add the omero_imageJ to the plugin. 
 * Add the BF-tools jar. If you have it installed, you will need to update it.

test 1:
* Open an image in ImageJ
* Under the plugin menu. Select OMERO>Upload and Save To OMERO.
* The data manager and the importer dialog should pop up.
* Add the file to the queue and import

test 2:
* Open an image in ImageJ. Crop it for example
* Under the plugin menu. Select OMERO>Upload and Save To OMERO.
* The data manager and the importer dialog should pop up.
* Add the file to the queue and import. An ome-tiff will be generated and imported.


ROIs are not yet supported for test 1
